packages/viewer/src/components/viewer-markup-arrow/utils.ts

```diff
@@ -0,0 +1,56 @@
+import { Angle, Matrix, Point } from '@vertexvis/geometry';
+
+export interface ArrowheadPoints {
+  tip: Point.Point;
+  left: Point.Point;
+  right: Point.Point;
+  base: Point.Point;
+}
+
+export function createArrowheadPoints(
+  start: Point.Point,
+  end: Point.Point,
+  arrowAngle = 60
+): ArrowheadPoints {
+  const distance = Point.distance(start, end);
+  const angle = Angle.normalize(
+    Angle.toDegrees(Angle.fromPoints(start, end)) - 270
+  );
+
+  const height = Math.max(4, Math.min(16, distance * 0.25));
```

Comment:
What is this height accounting for? the `sideLength`? etc. Some notes/comments on this might not hurt to add to add above the method to help our future self. 😄

Reply:
It adjusts the size of the arrow head, based on how long the arrow is.